packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -6,6 +6,146 @@ section: get-started
+
+### React 19 support
+
+PatternFly now includes support for React 19 across all components and extensions, enabling you to benefit from  new React features and performance enhancements.
+
+This update is non-breaking, and PatternFly 6 will continue to support React 17 and 18 in line with our support policy. In future releases, we plan to more fully leverage React 19's capabilities to enhance existing PatternFly features and introduce new ones.
+
+### Component animations
+
+As we progress in our effort to integrate motion design across PatternFly components, we added the following new animations in this release. To provide a holistic view of our animated components, we also put together [an interactive animations demo](/design-foundations/motion/react-demos), which we will continue to update as we add additional animation support.   
+
+#### Expansion 
+
+We added expansion animations that apply a fade-in and slide-in transition as hidden content sections are expanded in a variety of components. A similar, reversed animation applies when expanded sections are collapsed.
+
+The following list highlights opt-in requirements and links to examples. Unless an animation is marked with "Requires opt-in," it will be enabled by default. 
+
+- Accordion 
+    - **Example:** All [accordions](/components/accordion) in our examples and demos.
+
+- Dual list selector 
+    - **Requires opt-in.**
+    - **Example:** [Dual list selector with tree](/components/dual-list-selector#with-tree), which has opted in.
+
+- Expandable section 
+    - **Example:** All [expandable sections](/components/expandable-section) in our examples and demos.
+
+- Form
+    - **Requires opt-in.**
+    - **Example:** [Form field groups](/components/forms/form#field-groups), which have opted in.  
+
+- Navigation
+    - **Example:** [Expandable navigation examples](/components/navigation#expandable).
+    - **Note:** This animation was introduced in our 6.2 release, but performance issues were reported. For this release, we reworked the animation to resolve these issues.
+
+- Search input   
+    - **Example:** [Expandable search input](/components/search-input#with-expandable-button).
+
+- Tree view
+    - **Requires opt-in.**
+    - **Example:** All [tree view](/components/tree-view) examples and demos, which have opted in. 
+
+- Table
+    - **Requires opt-in.**
+    - **Example:** [Expandable table](/components/table#expandable), which has opted in.
+    - **Note:** This release does not include expansion animations for tree tables.
+
+#### Feedback 
+
+The following components contain animations that give users dynamic feedback, helping to communicate that an action has taken or will take place.  
+
+1.  Buttons
+    - **Animation:** When a button is clicked, the active state color ripples outward from the center of the button.
+    - **Example:** All buttons in our examples and demos, including our [animated favorite button example](/components/button#favorite).
```

Comment:
This doesn't go to the favorite button example (maybe it works when deployed?). Anyway, I think it would be good to separate the button click ripple from the favorite button star pulse. We talk about the favorite button below, so maybe here we just call out the ripple effect when (most) any button is clicked.

Reply:
oh great point, ty! I don't think I meant to link there